src/VisualStudio/CSharp/Impl/LanguageService/CSharpDebuggerIntelliSenseContext.cs

```diff
@@ -102,18 +102,18 @@ protected override IProjectionBuffer GetAdjustedBuffer(int contextPoint, Documen
                 // where [] denotes CurrentStatementSpan, should use the start of CurrentStatementSpan
                 // as the adjusted context, and should not place a semicolon before debuggerMappedSpan.
                 // Not doing either of those would place debuggerMappedSpan outside the for loop.
-                separatorBeforeDebuggerMappedSpan = string.Empty;
+                separatorBeforeDebuggerMappedSpan = " ";
```

Comment:
Add a comment explaining that whitespace is preferred over empty string, because empty string causes the completion trigger point to map to an incorrect snapshot

Reply:
Done!